.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for AKS Engine
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the request**
+
+**Describe the solution you'd like**
+
+**Describe alternatives you've considered**
+
+**Additional context**
```

Comment:
I'd add something like "why is this needed" , ie. convince us that we want this

Reply:
That's important, so I added it as the second section in feature request template.